From dfa9fa931f68ba9a5247497c0796ca663fde0e8a Mon Sep 17 00:00:00 2001
From: zhaochengyi <zhaochengyi@uniontech.com>
Date: Thu, 3 Nov 2022 19:24:15 +0800
Subject: [PATCH] Set volume with ignored dB for Philips SPA2100
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The sound card PCM dB range of the device is
-128 to 128. If dB is sent to the sound card,
the volume will be abnormal. so ignore the dB
and set the volume directly to the sound card.

Log: 该设备的声卡PCM dB范围为-128到128, 如果
设置dB给声卡，会出现音量异常，因此忽略dB，
直接设置volume给声卡。

Bug: https://pms.uniontech.com/bug-view-122957.html

Author: Chengyi Zhao <zhaochengyi@uniontech.com>
Origin: https://gerrit.uniontech.com/plugins/gitiles/base/pulseaudio/+/bdaa155eddd3ab4f251a37e82f869675c56cd2c2
---
 src/modules/alsa/90-pulseaudio.rules | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/modules/alsa/90-pulseaudio.rules b/src/modules/alsa/90-pulseaudio.rules
index 0ae28fc..e2cabb1 100644
--- a/src/modules/alsa/90-pulseaudio.rules
+++ b/src/modules/alsa/90-pulseaudio.rules
@@ -178,6 +178,8 @@ ATTRS{idVendor}=="0bda", ATTRS{idProduct}=="4f05", ENV{PULSE_PROFILE_SET}="dahua
 # For ignore_dB
 # Huawei
 ATTRS{idVendor}=="31b2", ATTRS{idProduct}=="0011", ENV{PULSE_VOLUME_IGNORE_DB}="1"
+# Philips SPA2100
+ATTRS{idVendor}=="e5b7", ATTRS{idProduct}=="0811", ENV{PULSE_VOLUME_IGNORE_DB}="1"
 
 GOTO="pulseaudio_end"
 
-- 
2.33.1

